images/nginx/Makefile

```diff
@@ -14,9 +14,9 @@
 
 # 0.0.0 shouldn't clobber any released builds
 TAG ?= 0.39
-REGISTRY ?= quay.io/kubernetes-ingress-controller
+REGISTRY ?= shopify
 ARCH ?= $(shell go env GOARCH)
-DOCKER ?= gcloud docker --
+DOCKER ?= docker
```

Comment:
can you avoid changing Makefile's and set these environment variables in `travis.yml`?